tests/acceptance/pageObjects/FilesPageElement/publicLinksDialog.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
 
       return this.click({ locateStrategy: 'xpath', selector: createLinkButtonXpath })
         .waitForOutstandingAjaxCalls()
-        .waitForElementNotPresent({ locateStrategy: 'xpath', selector: createLinkButtonXpath })
+        .waitForElementNotPresent({ locateStrategy: 'xpath', selector: createLinkButtonXpath, abortOnFailure: false })
```

Comment:
do not fail the function when the button is not gone in time, that way the same function can be used for steps that expect the sharing not to work